components/pages/header.vue

```diff
@@ -1,13 +1,229 @@
 <template>
-  <h1>header section</h1>
+  <div>
+    <header>
+      <a href="#" class="logo"><font-awesome-icon :icon="['fas', 'utensils']" />resto.</a>
+
+      <nav class="navbar" :class="{ active: isActive }">
+        <a v-for="(item, index) in menuList" :key="index" :href="item.href" :class="{ 'active': index === itemActive }">{{ item.label }}</a>
+      </nav>
+      <div class="icons">
+        <font-awesome-icon class="items menu-mobile" :icon="['fas', isActive ? 'times' : 'bars']" @click="isActive = !isActive" />
+        <font-awesome-icon class="items" :icon="['fas', 'magnifying-glass']" @click="isToggleSearch = !isToggleSearch" />
+        <font-awesome-icon class="items" :icon="['fas', 'heart']" />
+        <font-awesome-icon class="items" :icon="['fas', 'cart-shopping']" />
+      </div>
+    </header>
+    <form class="search-form" :class="{active: isToggleSearch}" action="">
+      <input class="search-box" type="search" name="" placeholder="Search here....">
+      <label for="search-box">
+        <font-awesome-icon :icon="['fas', 'magnifying-glass']" />
+      </label>
+      <font-awesome-icon class="close" :icon="['fas', 'xmark']" @click="isToggleSearch = !isToggleSearch" />
+    </form>
+  </div>
 </template>
 
 <script>
 export default {
-  name: 'PageHeader'
+  name: 'PageHeader',
+  data () {
+    return {
+      isActive: false,
+      isToggleSearch: false,
+      itemActive: 0,
+      menuList: [
+        {
+          id: 1,
+          href: '#home',
+          label: 'Home'
+        },
+        {
+          id: 2,
+          href: '#dishes',
+          label: 'Dishes'
+        },
+        {
+          id: 3,
+          href: '#about',
+          label: 'About'
+        },
+        {
+          id: 4,
+          href: '#menu',
+          label: 'Menu'
+        },
+        {
+          id: 5,
+          href: '#review',
+          label: 'Review'
+        },
+        {
+          id: 6,
+          href: '#order',
+          label: 'Order'
+        }
+      ]
+    }
+  }
 }
 </script>
 
 <style lang="scss" scoped>
+  header {
+    position: fixed;
+    top: 0; left: 0; right: 0;
+    background-color: #fff;
+    padding: 1rem 7%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 1000;
+    box-shadow: var(--box-shadow);
```

Comment:
we using scss, so i suggest to using the scss variables, plz reference at below link:
https://sass-lang.com/documentation/variables